cmd/policy-attributeValues.go

```diff
@@ -127,11 +127,11 @@ func policy_deactivateAttributeValue(cmd *cobra.Command, args []string) {
 }
 
 func policy_unsafeReactivateAttributeValue(cmd *cobra.Command, args []string) {
-	h := NewHandler(cmd)
+	c := cli.New(cmd, args)
+	h := NewHandler(c)
 	defer h.Close()
```

Comment:
We're definitely not always deferring a close of the handler. It [looks like](https://github.com/opentdf/otdfctl/blob/e0bc1836e8b5f14c87b5d572ad7937924c76d860/pkg/handlers/sdk.go#L108) it calls `sdk.Close()`. Is that something we need to always be doing?

Reply:
Not critical right now. What I'd like to do in the next refactor is move `NewHandler` and `InitProfile` into the `cli.New()` process... I feel like we can handle the close as part of `c.Close()` and make sure it's uniform.

I think part of that work will be adding `cli.New()` to the `preRun` process. Not sure if we can pass the object back to the command but something to look at later.

Reply:
That seems like a nice improvement. It looks like there's [a flag we can set in Cobra](https://github.com/spf13/cobra/blob/78bfc837fe358c750faa7e7f0a8016b300044d58/site/content/user_guide.md?plain=1#L698C6-L698C28) to execute a parent's prerun hook on all child commands (so we shouldn't need to write some kind of recursive init function ourselves).

Reply:
Created an issue #354